frontend/src/APIClients/ResidentAPIClient.ts

```diff
@@ -0,0 +1,29 @@
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+import baseAPIClient from "./BaseAPIClient";
+
+import { Resident } from "../types/ResidentTypes";
+
+const getResidents = async (): Promise<any> => {
```

Comment:
Like above, getting rid of this of this any would be nice

Reply:
Copying over types from https://github.com/uwblueprint/supportive-housing/pull/142 ; resolved